src/packages/migrate/src/utils/handlePanic.ts

```diff
@@ -0,0 +1,90 @@
+import chalk from 'chalk'
+import { RustPanic, sendPanic, link } from '@prisma/sdk'
+import isCi from 'is-ci'
+import prompt from 'prompts'
+
+export async function handlePanic(
+  error: RustPanic,
+  cliVersion: string,
+  binaryVersion: string,
+): Promise<boolean> {
+  return new Promise(async function (resolve, reject) {
+    if (!process.stdout.isTTY || isCi || process.env.GITHUB_ACTIONS) {
+      return reject(error)
+    }
+
+    await panicDialog(error, cliVersion, binaryVersion)
+
+    return resolve()
+  })
+}
+
+async function panicDialog(error, cliVersion, binaryVersion) {
+  const errorMessage = error.message
+    .split('\n')
+    .slice(0, process.stdout.rows - 20)
```

Comment:
I would be careful with this, it could be, that users just have a terminal size of 5 rows, because they might just have it small in that moment.
You might want to set a minimum with `Math.max(min, ...)`

Reply:
I just copy pasted what was done in ink here

Reply:
I can improve it for sure